substrate/primitives/runtime-interface/src/lib.rs

```diff
@@ -17,7 +17,6 @@
 
 // Custom inner attributes are unstable, so we need to faky disable the attribute.
 // rustfmt still honors the attribute to not format the rustdocs below.
-#![cfg_attr(feature = "never", rustfmt::skip)]
```

Comment:
Did you read the comment above? :D Not sure this is already fixed in rustfmt?

Reply:
sorry I missed it, but I did run `cargo fmt -p sp-runtime-interface` the file was unchanged so I thought it wasn't relevant anymore.

Reply:
The CI uses a specific nightly for formatting: `cargo +nightly-2024-04-14 fmt`.

Reply:
Also tested with this version, anyway CI should tell if the formatting of the crate is changing with this PR, in this case I'll revert this change.